src/source/publishing/publisher.cr

```diff
@@ -27,7 +27,11 @@ module PlaceOS::Source
     private def consume_messages
       spawn do
         while message = message_queue.receive?
-          publish(message)
+          begin
+            publish(message)
+          rescue error
+            Log.warn(exception: error) { "publishing message: #{message}" }
```

Comment:
Is there a chance sensitive data could be logged here?

Reply:
nah, anything being published here is already exposed as state which is visible to any user

Reply:
plus we want to know what is causing errors so we can fix it